Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -62,9 +67,26 @@ void ITSTrackTask::initialize(o2::framework::InitContext& /*ctx*/)
   mNtracksMAX = std::stof(mCustomParameters["NtracksMAX"]);
   mDoTTree = std::stoi(mCustomParameters["doTTree"]);
   nBCbins = std::stoi(mCustomParameters.find("nBCbins")->second);
+  mDictPath = mCustomParameters["clusterDictionaryPath"];
 
   createAllHistos();
   publishHistos();
+
+  std::ifstream file(mDictPath.c_str());
+
+  if (file.good()) {
```

Comment:
Please rebase with the master because this part has changed. Dictionary is fetched from CCDB now.